torch/utils/cpp_extension.py

```diff
@@ -2206,7 +2206,7 @@ def is_ninja_available():
 def verify_ninja_availability():
     """Raise ``RuntimeError`` if `ninja <https://ninja-build.org/>`_ build system is not available on the system, does nothing otherwise."""
     if not is_ninja_available():
-        raise RuntimeError("Ninja is required to load C++ extensions")
+        raise RuntimeError("Ninja is required to load C++ extensions (pip install ninja to get it)")
```

Comment:
maybe just `install ninja` could be through apt, conda, uv, etc...

Reply:
Also shouldn't this be an importerror?

Reply:
pip is the common way to get it from here I think.
I kept the error type from before, I can change in a follow up if you really want to!